### PR TITLE
Require PR title to include typed emoji

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:
This PR simplifies the PR template and requires users to add an emoji to the title to generate release notes

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ If you're new to the project and want to help, but don't know where to start, we
 1. If you haven't already done so, sign a Contributor License Agreement (see details above).
 1. Fork the desired repo, develop and test your code changes.
 1. Submit a pull request.
+  1. All code PR must be labeled with
+    ⚠️ (:warning:, major), ✨ (:sparkles, minor), 🐛 (:bug:, patch), 📖 (:book:, docs), or 🏃 (:running:, other)
```
Comment (detiber) [inline CONTRIBUTING.md]:
```suggestion
    ⚠️ (:warning:, major), ✨ (:sparkles:, minor), 🐛 (:bug:, patch), 📖 (:book:, docs), or 🏃 (:running:, other)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,20 +1,11 @@
-<!--  Thanks for sending a pull request!  Here are some tips for you:
-1. If this is your first time, read our contributor guidelines https://git.k8s.io/community/contributors/guide/pull-requests.md#the-pull-request-submit-process and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
-2. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
-3. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
-4. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
-5. If this PR changes image versions, please title this PR "Bump <image name> from x.x.x to y.y.y."
--->
+<!-- please add a icon to the title of this PR (see VERSIONING.md), and delete this line and similar ones -->
+<!-- the icon will be either ⚠️ (:warning:, major), ✨ (:sparkles, minor), 🐛 (:bug:, patch), 📖 (:book:, docs), or 🏃 (:running:, other) -->
```
Comment (detiber) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
<!-- the icon will be either ⚠️ (:warning:, major), ✨ (:sparkles:, minor), 🐛 (:bug:, patch), 📖 (:book:, docs), or 🏃 (:running:, other) -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,20 +1,11 @@
-<!--  Thanks for sending a pull request!  Here are some tips for you:
-1. If this is your first time, read our contributor guidelines https://git.k8s.io/community/contributors/guide/pull-requests.md#the-pull-request-submit-process and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
-2. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
-3. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
-4. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
-5. If this PR changes image versions, please title this PR "Bump <image name> from x.x.x to y.y.y."
--->
+<!-- please add a icon to the title of this PR (see VERSIONING.md), and delete this line and similar ones -->
+<!-- the icon will be either ⚠️ (:warning:, major), ✨ (:sparkles:, minor), 🐛 (:bug:, patch), 📖 (:book:, docs), or 🏃 (:running:, other) -->
```
Comment (detiber) [inline .github/PULL_REQUEST_TEMPLATE.md]: When does one use each of the icons? It's not overly clear from the documentation.

Comment (vincepri) [reply .github/PULL_REQUEST_TEMPLATE.md]: Added some more directions, wdyt?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,20 +1,11 @@
-<!--  Thanks for sending a pull request!  Here are some tips for you:
-1. If this is your first time, read our contributor guidelines https://git.k8s.io/community/contributors/guide/pull-requests.md#the-pull-request-submit-process and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
-2. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
-3. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
-4. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
-5. If this PR changes image versions, please title this PR "Bump <image name> from x.x.x to y.y.y."
--->
+<!-- please add a icon to the title of this PR (see VERSIONING.md), and delete this line and similar ones -->
+<!-- the icon will be either ⚠️ (:warning:, major or breaking changes), ✨ (:sparkles:, minor or feature additions), 🐛 (:bug:, patch and bugfixes), 📖 (:book:, documentation or proposals), or 🏃 (:running:, other) -->
```
Comment (detiber) [inline .github/PULL_REQUEST_TEMPLATE.md]: lgtm, can you also update CONTRIBUTING.md?

**Pull request merged**